src/components/CityCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+function CityCard({
+  name, continent, flag, lat, lon,
+}) {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const handleNavigation = (city) => {
+    navigate(`${city}`, {
+      state: {
+        name, continent, lat, lon,
+      },
+    });
+  };
+  return (
+    <div className="city-card" onClick={() => handleNavigation('city')}>
+      <h4>{name}</h4>
+      <p>
+        lat :
+        {lat}
+      </p>
+      <img src={flag} alt="country-flag" />
+    </div>
+  );
+}
```

Comment:
- [ ] According to requirements you should stick to the design and layouts, so it would be better to make the cards the same as the design by removing the black borders and spaces between cards as in the design 
- Yours 
<img width="338" alt="Screenshot 2023-06-03 at 17 00 46" src="https://github.com/LDouglasOT/AirPollution/assets/61790587/4d86a0cd-7bf7-422d-9fd6-ff6939719371">

- Design 

<img width="263" alt="Screenshot 2023-06-03 at 17 00 05" src="https://github.com/LDouglasOT/AirPollution/assets/61790587/256bcfae-6259-414e-b3a6-45849db9a0e6">

- Screenshot of the requirements 
<img width="582" alt="Screenshot 2023-06-03 at 17 06 39" src="https://github.com/LDouglasOT/AirPollution/assets/61790587/8a2bb4bb-c4d3-4bc4-9a9f-ac023e7cbaf8">

Reply:
fixed